### DISCUSSION TITLE
Add `--password-stdin` option to `gh valet update`.

### DISCUSSION THREAD

**Discussion by korosuke613** [github/gh-valet#38] [Ideas]
`gh valet update` can specify `--password`, but this method is insecure. (I think this warning is issued by `docker login`.)

```console
❯ gh valet update --username korosuke613 --password ***
WARNING! Using --password via the CLI is insecure. Use --password-stdin.
...
```

But, `gh-valet update` does not have the `--password-stdin` option.

```console
❯ gh valet update --username korosuke613 --password-stdin
Unrecognized command or argument '--password-stdin'.
...
```

So I would like to see the `--password-stdin` option add to `gh valet update`.

---

GitHub CLI version: 2.11.3 (2022-05-25)
gh-valet version: v0.1.9

**Comment by ethanis** [comment]
Agreed that this would be a nice improvement! A `--password-stdin` option was added in the [v0.1.10](https://github.com/github/gh-valet/releases/tag/v0.1.10) release.

**Reply by korosuke613** [reply]
Thanks!!
